docs/001_develop/03_client-capabilities/008_interaction/button.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: 'Button'
+sidebar_label: 'Button'
+id: client-interaction-button
+keywords: [interaction, interactive, button, form, click, submit, data]
+tags:
+- interaction
+- interactive
+- button
+- form
+- click
+- change
+- submit
+- data
+sidebar_position: 8
+---
+
+import ButtonDemo from '../../../../examples/ui/client-capabilities/interaction/button.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Button
+
+Implementation of a [button](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/button) component.
+
+Use cases:
+* Submit forms
+* Trigger actions
+* Create interactive elements
+* Set options
+
+## Example
+
+<ButtonDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+  <TabItem value="genesis">
+
+    Declaration
+    ```html
+    <rapid-button>Button</rapid-button>
+    ```
+
+    Usage
+    ```typescript
+    @customElement({
+      name: 'my-element',
+      template: html`
+      <rapid-button @click=${x => x.buttonClicked()}>
+        Button
+      </rapid-button>
+    `,
+    })
+    export class MyElement extends GenesisElement {
+      buttonClicked() {
+        alert('Button Clicked')
+      }
+    }
+    ```
+
+  </TabItem>
+  <TabItem value="react">
+
+    Declaration
+    ```html
+    <rapid-button></rapid-button>
+    ```
+    Usage
+    ```typescript
+    export function MyComponent() {
+      const handleButtonClicked = (event) => {
+        console.log('Button clicked')
+      };
+
+      return (
+        <rapid-button onClick={handleButtonClicked}></rapid-button>
+      );
+    }
+    ```
+  </TabItem>
+  <TabItem value="angular">
+
+    Declaration
+    ```html
+    <rapid-button></rapid-button>
+    ```
+
+    Usage
+    ```typescript
+    import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+    import { FormsModule } from '@angular/forms';
+    @Component({
+      selector: 'my-root',
+      template: `
+        <rapid-button
+          (click)="onButtonClicked($event)"
+        ></rapid-checkbox>
+      `,
+      styleUrls: ['./my.component.css'],
+      standalone: true,
+      schemas: [CUSTOM_ELEMENTS_SCHEMA],
+      imports: [FormsModule],
+    })
+    export class AppComponent {
```

Comment:
Tested and works in angular. Same as above - can we add the slots, appearance examples